clang/test/Analysis/cxx-uninitialized-object.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_analyze_cc1 -std=c++14 -verify  %s \
+// RUN: %clang_analyze_cc1 -std=c++14 -verify=expected,pedantic  %s \
```

Comment:
You introduced the `pedantic` verify prefix, but never actually used it.

Reply:
It's used like the `pedantic-warning`, I'll remove this to address previous comments.